app/assets/stylesheets/custom.css.scss

```diff
@@ -1,3 +1,195 @@
 body {
   padding-top: 50px;
 }
+
+.bg-light-gray {
+    background-color: #f7f7f7;
```

Comment:
Color literals like `#f7f7f7` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces